### PR TITLE
Add Lets Do This

### LINKED ISSUE #8175: Add Lets Do This
### Site name

Lets Do This

### Site URL

https://www.letsdothis.com

### Documentation URL

https://intercom.help/ldt-partner-help/en/articles/9423487-two-factor-authentication

### Recovery Documentation URL

_No response_

### Supported 2FA methods

- [X] TOTP (RFC-6238, Google Authenticator)
- [ ] Proprietary Software (Authy, Duo, etc.)
- [ ] U2F/WebAuthn security keys
- [ ] Proprietary hardware (Yubico OTP, RSA keys, etc.)
- [ ] Phone calls
- [ ] SMS tokens
- [ ] Email tokens

### Regions served

_No response_

### Eligibility

- [X] I have checked that the site meets 2fa.directory's [contributing guidelines and site criteria](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md)
- [X] The site is ranked within the top 200,000 global sites on [Similarweb](https://www.similarweb.com/).
- [X] The site does not contain or promote content that violates 2fa.directory's [excluded categories and websites guidelines](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md).
- [X] The issue I'm creating is not a duplicate of an existing [issue](https://github.com/2factorauth/twofactorauth/issues) or [pull request](https://github.com/2factorauth/twofactorauth/pulls).

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by haroldadmin**
Resolves #8175

### CODE REVIEW

File: img/l/letsdothis.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="80" height="80" fill="none"><path fill="#0F32FA" d="M0 0h80v80H0z"/><g clip-path="url(#a)"><path fill="#fff" d="m52.992 45.958.54-10.313-24.09-1.259L64 32.578 63.45 22 16 24.486l.55 10.577 10.458-.55-.54 10.324 23.91 1.259-31.097 1.629.561 10.578 40.856-2.148-.55-10.577z"/></g><defs><clipPath id="a"><path fill="#fff" d="M16 22h48v36.303H16z"/></clipPath></defs></svg>
```
Comment (Carlgo11) [inline img/l/letsdothis.com.svg]: Small change to fix the viewbox warning.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 80 80" style="background:#0f32fa"><path fill="#fff" d="m53 46 .5-10.4-24-1.2L64 32.6 63.5 22 16 24.5l.6 10.6 10.4-.6-.5 10.3 23.9 1.3-31.1 1.6.5 10.6 40.9-2.1-.6-10.6"/></svg>
```

File: entries/l/letsdothis.com.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "LetsDoThis": {
+    "domain": "letsdothis.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://intercom.help/ldt-partner-help/en/articles/9423487-two-factor-authentication",
```
Comment (Carlgo11) [inline entries/l/letsdothis.com.json]:
```suggestion
    "documentation": "https://intercom.help/ldt-partner-help/en/articles/9423487",
```

File: entries/l/letsdothis.com.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "LetsDoThis": {
```
Comment (Carlgo11) [inline entries/l/letsdothis.com.json]: The official name seems to be this:
```suggestion
  "Let's Do This": {
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Thanks for the pull request @haroldadmin!
I've attached a few suggestions.

**Comment by haroldadmin** [comment]
Thanks for the suggestions @Carlgo11! This is ready for a re-review.

**Review by Carlgo11 — APPROVED** [review]
Thank you! 🙂
Your contribution will be visible in around 10 minutes. Sometimes, it takes longer to show up due to [server and browser caching](https://github.com/2factorauth/twofactorauth/wiki/FAQ-Caching).

**Pull request merged**